geomet/esri.py

```diff
@@ -25,9 +25,7 @@ def load(source_file):
          A GeoJSON `dict` representing the geometry read from the file.
 
     """
-    with open(source_file, 'r') as reader:
-        data = json.loads(reader.read())
-        return loads(data)
+    return json.load(source_file)
```

Comment:
You want to return `loads(json.load(source_file)`, right? because we want to read the content from the file and transform it from EsriJSON to GeoJSON.

Reply:
I'm still working on this

Reply:
thanks for pointing it out

Reply:
Sorry about that! I'll hold off for a minute =)

Reply:
ok please take a look

Reply:
`json.load` takes a stream, not a str. Suggest:
```
with open(source_file, 'r') as stream:
    return json.load(stream)
```